docs/inputs.md

```diff
@@ -159,8 +159,8 @@ The table below contains brief descriptions of the input files required to execu
 | remoteAVParking | Remote AV parking available at MGRA: 0 = Not available, 1 = Available | 
 | refueling_stations | Number of refueling stations at MGRA | 
 | MicroAccessTime | Micro-mobility access time (mins) | 
```

Comment:
@JiaXu1024 please also update MicroAccessTime description to be clearer. It looks like App got confused recently thinking that it is microtransit access time. Maybe the description could be "Shared micro-mobility (e-scooter/e-bike) access time (mins)"?

Reply:
tagging @cundo92 and @aber-sandag to see if they have any additional input